hr_attendance_management/models/hr_employee.py

```diff
@@ -24,12 +24,14 @@ class HrEmployee(models.Model):
              " only at the cron execution time.")
     annual_balance_date = fields.Date(
         compute='_compute_annual_balance_date')
+    current_period_start_date = fields.Date(
+        compute="_compute_current_period_start_date", store=False)
 
     attendance_days_ids = fields.One2many('hr.attendance.day', 'employee_id',
                                           "Attendance days")
     balance = fields.Float(compute='_compute_balance', store=True)
 
-    extra_hours_lost = fields.Float(compute='_compute_balance', store=True)
+    current_period_lost_hours = fields.Float(compute='_compute_balance', store=True)
```

Comment:
I would suggest that we keep the name `extra_hours_lost` to avoid migration issues and having to change it everywhere.

Reply:
Changed in next commit.